Jellyfin/ViewModels/OnBoardingViewModel.cs

```diff
@@ -87,17 +97,51 @@ public string ServerUrl
         }
     }
 
+    /// <summary>
+    /// Gets the list of discovered servers on the network.
+    /// </summary>
+    public ObservableCollection<DiscoveredServer> DiscoveredServers { get; } = new ObservableCollection<DiscoveredServer>();
+
     /// <summary>
     /// Gets the command to connect to the server.
     /// </summary>
     public IRelayCommand ConnectCommand { get; private set; }
 
+    /// <summary>
+    /// Gets the command to connect to the discovered server.
+    /// </summary>
+    public IRelayCommand<DiscoveredServer> ConnectToCommand { get; private set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether server discovery is in progress.
+    /// </summary>
+    public bool DiscoveryInProgress
+    {
+        get
+        {
+            return field;
+        }
+
+        set
+        {
+            SetProperty(ref field, value);
+        }
+    }
+
+    private void ServerDiscovery_OnDiscoveryEnded()
```

Comment:
Please don't use underscore for the method naming. It feels out of place in this codebase/ C#.